src/views/Login.tsx

```diff
@@ -124,7 +125,10 @@ export default function Login() {
             className="us-none pointer">
             <span className="underline">Offline</span> ↗
           </Flex.Item>
-          <Flex.Item className="gray4">v{version}</Flex.Item>
+          <Flex.Item className="gray4">
+            v{version}
+            {isRopsten ? 'r' : ''}
+          </Flex.Item>
```

Comment:
This is good, but is there a way to modify the `version` directly, rather than just affecting its displayed value? We also use it in the `zip` command, as the filename.

(This isn't actually a big deal, considering I never build .zip releases for Ropsten, but just wondering if there's a higher-level approach here.)

Reply:
Good thinking, just made a few tweaks to allow this value to be set via an env var (`REACT_APP_BRIDGE_VERSION`); it will fallback to the version from `package.json` if unset. And, it will use this value for the UI and zip filename.